src/javascripts/components/copy.js

```diff
@@ -7,15 +7,19 @@
   // This module is dependent on /vendor/clipboard.js
   GOVUK.copy = {
     init: function (selector) {
-      $(selector).parent().prepend('<a class="govuk-link app-c-link--copy" href="#copy">Copy</a>')
+      $(selector).parent().prepend('<a class="govuk-link app-c-link--copy" href="#copy" aria-live="assertive">Copy</a>')
```

Comment:
Does aria-live need to be injected like this?

Reply:
the link is injected

Reply:
Sorry, I didn't read the diff properly :man_facepalming: